package.json

```diff
@@ -62,7 +62,8 @@
     "@remix-run/react": "2.8.0",
     "@remix-run/server-runtime": "2.8.0",
     "@sentry/profiling-node": "^7.105.0",
-    "@sentry/remix": "^7.105.0",
+    "@sentry/remix": "^7.107.0",
+    "@sentry/vite-plugin": "^2.15.0",
```

Comment:
noticing this now but @sentry/vite-plugin should be in dev deps!? @devlinmorrow

Reply:
Yes, I believe we could move this to dev deps. PR welcome!